src/lib.rs

```diff
@@ -96,14 +89,14 @@ impl UPClientZenoh {
     /// #[tokio::main]
     /// # async fn main() {
     /// use up_transport_zenoh::{zenoh_config, UPClientZenoh};
-    /// let upclient = UPClientZenoh::new(zenoh_config::Config::default(), String::from("MyAuthName"))
+    /// let upclient = UPClientZenoh::new(zenoh_config::Config::default(), "MyAuthName")
     ///     .await
     ///     .unwrap();
     /// # }
     /// ```
     pub async fn new(
         config: zenoh_config::Config,
-        authority_name: String,
+        authority_name: impl Into<String>,
```

Comment:
Did you run into a case where this was needed? Just my own curiosity :slightly_smiling_face:

Reply:
No, just found that I can make the code more general by accepting `impl Into<String>`. We can accept `String`, `&String`, `&str` here. Therefore, if you want have `UPClientZenoh::new()`, you don't always need `String::from("authority")`, but just only put `"authority"` here.